internal/commands/upgrade.go

```diff
@@ -55,26 +55,32 @@ func runUpgrade(dockerCli command.Cli, installationName string, opts upgradeOpti
 		}
 		c.Bundle = b
 	}
-	driverImpl, err := prepareDriver(dockerCli)
+	c.Parameters, err = mergeBundleParameters(c.Bundle,
+		withFileParameters(opts.parametersFiles),
+		withCommandLineParameters(opts.overrides),
+	)
 	if err != nil {
 		return err
 	}
-	creds, err := prepareCredentialSet(targetContext, dockerCli.ContextStore(), c.Bundle, opts.credentialsets)
+	var specifiedOrchestrator string
+	if rawOrchestrator, ok := c.Parameters["docker.orchestrator"]; ok {
```

Comment:
Nothing to do with this PR, but I wonder if we should forbid the overriding of the `docker.orchestrator` parameter during an `upgrade` ? @simonferquel ?